deb-package/build-deb.sh

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env bash
+set -xe
+
+# cleanup previous installations
+rm -rf deb
+
+# create binaries dir
+INSTALL_DIR="deb/opt/dedis/dvoting/bin"
+mkdir -p $INSTALL_DIR
+
+# copy binaries to deb/opt/dedis/dvoting/bin
+UKAPP_DIR="../contracts/evoting/unikernel/apps/combine_shares"
+cp $UKAPP_DIR/create-bridge $INSTALL_DIR
+cp $UKAPP_DIR/start-unikernel $INSTALL_DIR
+cp $UKAPP_DIR/qemu-guest $INSTALL_DIR
+cp $UKAPP_DIR/build/combine_shares_kvm-x86_64 $INSTALL_DIR
+
+DVOTING_CLI_DIR=".."
+cp $DVOTING_CLI_DIR/memcoin $INSTALL_DIR
+
+# add config files
+cp -a pkg/etc deb
+cp -a pkg/lib deb
+cp -a pkg/opt deb
+cp -a pkg/var deb
+
+# add folders
+mkdir -p deb/var/log/dedis/dvoting
+
+# adjust permissions
+find deb ! -perm -a+r -exec chmod a+r {} \;
+
+# get version from git without v prefix
+GITVERSION=$(git describe --abbrev=0)
+VERSION=${GITVERSION:1}
+if [[ -z "${ITERATION}" ]]
+then
+  ITERATION="0"
+fi
+
+# fpm needs an existing output directory
+OUTPUT_DIR="dist"
+mkdir -p $OUTPUT_DIR
+
+fpm \
+    --force -t deb -a all -s dir -C deb -n dedis-dvoting -v ${VERSION} \
+    --iteration ${ITERATION} \
+    --deb-user dvoting \
+    --deb-group dvoting \
+    --depends bridge-utils \
```

Comment:
we also depend on `bash` being present. 
Unless we don't *really* need bash and we can just rely on `/bin/sh` in our scripts ?